client/app/components/guides/guides.less

```diff
@@ -1,13 +1,7 @@
-@borderColor: #e1e4df;
-@textColor: #676767;
-
-.lightbulb-guide {
-    button {
-        background-color: transparent;
-        color: @textColor;
-        padding: .5rem 0;
-        text-align: left;
-        white-space: normal;
-        width: 100%;
+.oui-tile {
```

Comment:
Does it will be backported to OVH UI Kit? If yes, could you please adding a comment such as "TODO: should be removed when it will be available on the OVH UI Kit"

Reply:
Pretty sure that most "TODO" are just comments that will be forgotten in near future

Reply:
That's your opinion but a "TODO" comment is always appreciable if in a near future someone goes through here AND/OR decide to tackle some TODOs.